content/en/APIUseCases/create_asset.md

```diff
@@ -0,0 +1,229 @@
+---
+title: "Use Our API to Create an Asset"
```

Comment:
Option: Create an Asset via the API

Since we'll add more guides to this section, I would start the title with an action and then add "Using the API" or "via the API." At the same time, if we repeat it in each guide, this will be redundant.

Reply:
> At the same time, if we repeat it in each guide, this will be redundant.

Good point. Since this book already has API in the name, I'm going to limit such titles -- in this case "Create an Asset"

btw, the Google Developer Style guide suggests that we should not use `via`